style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.wrapper {
+  background-image: url(/images/header-background-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  min-height: 720px;
+  border-bottom-left-radius: 100px;
+}
+
+ul {
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 78px;
+  margin: 0 24px;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;

```

Comment:
- [ ] [OPTIONAL] At the moment, it is `san-serif` font that is displaying. You should use the `Poppins` which can be added by importing from google fonts